### DISCUSSION TITLE
Roll back button  :-)

### DISCUSSION THREAD

**Discussion by Doublejay** [transmission/transmission#4976] [Ideas]
thanks to everyone's efforts on Transmission andt to all devs in opensource.

I wonder if anyone would like to have a Roll Back button for upgrades ?  Or something similar ?

I was on autopilot this morning when i accepted an update to V4 - only to find it's not yet been "vetted"   by my trackers (like their going to actually look at the code...🤷🏼‍♂️)  .  Then I came and saw some of the issue with the Mac Ventura versions and it looks like I'm best served by rolling it back for the moment.

**Comment by ile6695** [comment]
It's really common for software to support only upgrades. Some do support cases where upgrade fails and software stays on the previous version, but that's not the same as what you suggest.
The best way to do a rollback for program that does not have an automated way to rollback is to first fully uninstall the program and then install the previous version. This way there are no configuration files that are not supported on the previous version of the program that were introduced with newer version.

**Comment by Doublejay** [comment]
Yep - I think I am being lazy.

What do you think about an option to report a different version of the client to the tracker ?

**Reply by ile6695** [reply]
This is what browsers do when reporting what features they support. It's a delicate mess.

I think it's better to be honest when reporting a version, because if trackers find out you're lying they could just completely block you.

**Reply by ckerr** [reply]
> What do you think about an option to report a different version of the client to the tracker ?

No. Never.

**Comment by Doublejay** [comment]
I am sure you are right. !

I will have to install another version for now.   

Thanks for your help.

**Reply by GaryElshaw** [reply]
Keep in mind that 4.0.1 will be out in the next day or two.

**Comment by Doublejay** [comment]
Sure... I’m not worried.   It’s   just that I’m at the mercy of the Tracker to decide when they have reviewed it - which I doubt they will actually  do, but instead  rely on community consensus, which is on the eleventy  first of next month.
